mappings/net/minecraft/client/gui/hud/InGameHud.mapping

```diff
@@ -17,8 +21,13 @@ CLASS cnr net/minecraft/client/gui/hud/InGameHud
 	FIELD w hudBossBar Lcnz;
 	METHOD a draw (F)V
 	METHOD a addChatMessage (Liy;Lja;)V
+	METHOD a setOverlayMessage (Lja;Z)V
+	METHOD a setPlayingOverlay (Ljava/lang/String;)V
```

Comment:
Is this the best name?

Reply:
Specifically, what does it refer to? Jukeboxes?

Reply:
Yep. It's the "now playing: 13" label you get above the hotbar.

Reply:
setPlayingOverlay sounds wrong to me... it's either too vague or insufficiently so.